tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -352,110 +274,112 @@ def delete_device(self):
             device_name = self.DishLeafNodePrefix + argin[i]
 
             try:
-                device_proxy = PyTango.DeviceProxy(device_name)
-                device_proxy.command_inout("SetStowMode")
-            except Exception as e:
-                print "Unexpected error in executing STOW command on ", device_name
-                self._read_activity_message = "Unexpected error in executing STOW command on " + str(device_name)
-                print "Error message is: \n", e
-                self._read_activity_message = "Error message is: \n" + str(e)
-                self.devlogmsg("Unexpected error in executing STOW command!", 2)
-
-        # PROTECTED REGION END #    //  CentralNode.StowAntennas
+                device_proxy = tango.DeviceProxy(device_name)
+                device_proxy.command_inout(CONST.CMD_SET_STOW_MODE)
+            except Exception as except_occured:
+                print CONST.ERR_EXE_STOW_CMD, device_name
+                self._read_activity_message = CONST.ERR_EXE_STOW_CMD + str(device_name)
+                print CONST.STR_ERR_MSG, except_occured
+                self._read_activity_message = CONST.STR_ERR_MSG + str(except_occured)
+                self.devlogmsg(CONST.STR_ERR_MSG, 2)
+        # PROTECTED REGION END #    //  CentralNode.stow_antennas
 
     @command(
     )
     @DebugIt()
     def StandByTelescope(self):
         # PROTECTED REGION ID(CentralNode.StandByTelescope) ENABLED START #
-        self.devlogmsg("StandByTelescope command invoked from Central node", 4)
-        self._read_activity_message = "StandByTelescope command invoked from Central node"
+        """ This command is to bring the Telescope into a STANDBY state (i.e. Low Power State) """
+        self.devlogmsg(CONST.STR_STANDBY_CMD_ISSUED, 4)
+        self._read_activity_message = CONST.STR_STANDBY_CMD_ISSUED
 
-        for name in range (0,len(self._dish_leaf_node_devices)):
+        for name in range(0, len(self._dish_leaf_node_devices)):
             try:
-                self._leaf_device_proxy[name].command_inout("SetStandbyLPMode")
-            except Exception as e:
-                print "Unexpected error in setting Standby mode on ", self._dish_leaf_node_devices[name]
-                self._read_activity_message = "Unexpected error in setting Standby mode on " + str(self._dish_leaf_node_devices[name])
-                print "Error message is: \n", e
-                self._read_activity_message = "Error message is: \n" + str(e)
-                self.devlogmsg("Unexpected error in executing StandByTelescope command!", 2)
-
-        # PROTECTED REGION END #    //  CentralNode.StandByTelescope
+                self._leaf_device_proxy[name].command_inout(CONST.CMD_SET_STANDBY_MODE)
+            except Exception as except_occured:
+                print CONST.ERR_EXE_STANDBY_CMD, self._dish_leaf_node_devices[name]
+                self._read_activity_message = CONST.ERR_EXE_STANDBY_CMD \
```

Comment:
Splitting a line with '\' is not a good practice. It should be parenthesis.

Reply:
Ensure line length of 110 characters and split the line with parenthesis if exceeding. Commit id is 172675f9b8198800667bd184e60043698176d4b3.